app/views/declaration/additional_fiscal_references.scala.html

```diff
@@ -83,17 +83,15 @@
 
     @components.error_summary(form.errors)
 
-    @components.section_header(messages("declaration.fiscalInformation.header"))
-
-    @components.page_title(Some(messages("declaration.additionalFiscalReferences.title")))
```

Comment:
Can `page_title` be deleted or is the component still used? Similar for `section_header`

Reply:
Yes good point I was thinking about that either! :stuck_out_tongue: